app/electron/main.ts

```diff
@@ -319,6 +344,30 @@ function getDefaultAppMenu(): AppMenu[] {
           role: 'forcereload',
           id: 'original-force-reload',
         },
+        {
+          label: i18n.t('Go To Home'),
+          role: 'homescreen',
+          id: 'original-home-screen',
+          click: () => {
+            mainWindow?.loadURL(startUrl);
+          },
+        },
+        {
+          label: i18n.t('Go Back'),
+          role: 'back',
+          id: 'original-back',
+          click: () => {
+            mainWindow?.webContents.goBack();
```

Comment:
Check if the webContents can go back before you call this function.

Reply:
done